nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -858,6 +858,8 @@
 
 - `qgis` and `qgis-ltr` are now built without `grass` by default. `grass` support can be enabled with `qgis.override { withGrass = true; }`.
 
+- `freecad` is now customizable package, which allows to call `customize` and save its configuration and modules in nix-way.
```

Comment:
It needs a better rewording.

Reply:
I am not very good in English. Can you suggest something specific, please?

Reply:
```suggestion
- `freecad` now supports addons and custom configuration in nix-way, which can be used by calling `freecad.customize`.
```

Reply:
@BonusPlay thank you!